spark/src/main/scala/org/apache/spark/sql/delta/catalog/AbstractDeltaCatalog.scala

```diff
@@ -602,6 +601,16 @@ class AbstractDeltaCatalog extends DelegatingCatalogExtension
     }
   }
 
+  /**
+   * The table feature CatalogOwnedTableFeature was once renamed from an old name. In a transition
+   * period we need to translate the old table feature property name set by caller to new one.
+   * TODO: clean up once callers are migrated.
```

Comment:
Do we need to explain what are the version that we need to keep this for the compatibility issue ?